docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -103,7 +175,253 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use YAML file ``values.yaml`` with content
+
+.. code:: sh
+
+  values:
+    greeting: Hello
+    recipient: World
+    message: '{{ (greeting + " " + recipient) * repeat }}'
+    date: '{{ yyyymmdd }}'
+    repeat: 1
+    empty: 
+
+and supplemental YAML file ``config.yaml`` with content
+
+.. code:: sh
+
+  values:
+    greeting: Good Night
+    recipient: Moon
+    date: 
+    repeat: 2
+
+* Show the values in the input config file that have unfilled Jinja2 templates or empty keys:
+
+  .. code:: sh
+
+    $ uw config realize --input-file values.yaml --output-format yaml --values-needed
+    [2024-01-05T11:34:22]     INFO Keys that are complete:
+    [2024-01-05T11:34:22]     INFO     values
+    [2024-01-05T11:34:22]     INFO     values.greeting
+    [2024-01-05T11:34:22]     INFO     values.recipient
+    [2024-01-05T11:34:22]     INFO     values.message
+    [2024-01-05T11:34:22]     INFO     values.repeat
+    [2024-01-05T11:34:22]     INFO 
+    [2024-01-05T11:34:22]     INFO Keys that have unfilled Jinja2 templates:
+    [2024-01-05T11:34:22]     INFO     values.date: {{ yyyymmdd }}
+    [2024-01-05T11:34:22]     INFO 
+    [2024-01-05T11:34:22]     INFO Keys that are set to empty:
+    [2024-01-05T11:34:22]     INFO     values.empty
+
+* To realize the config to ``stdout``, a target output format must be explicitly specified:
+
+  .. code:: sh
+
+    $ uw config realize --input-file values.yaml --output-format yaml
+    values:
+      greeting: Hello
+      recipient: World
+      message: Hello World
+      date: '{{ yyyymmdd }}'
+      repeat: 1
+      empty: null
+
+  Shell redirection via ``|``, ``>``, et al may also be used to stream output to a file, another process, etc.
+
+* Existing values can be overwritten and Jinja2 variables can be filled in by appending supplemental files to the end of the argument:
```

Comment:
```suggestion
* Values in the primary input file can be overridden via one or more supplemental files specified as positional arguments, each overriding the last; or by environment variables, which have the highest precedence.
```
Then maybe show something like
```
% recipient=Sun uw config realize --input-file values.yaml --output-format yaml config.yaml 
values:
  greeting: Good Night
  recipient: Moon
  message: Good Night SunGood Night Sun
  repeat: 2
```
That introduces the use of environment variables.

Reply:
Ah, good point! Got it added, thank you!